src/main/java/io/xpring/ilp/IlpClient.java

```diff
@@ -0,0 +1,101 @@
+package io.xpring.ilp;
+
+import org.interledger.spsp.server.grpc.CreateAccountResponse;
+import org.interledger.spsp.server.grpc.GetAccountResponse;
+import org.interledger.spsp.server.grpc.GetBalanceResponse;
+import org.interledger.spsp.server.grpc.SendPaymentResponse;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.xpring.xrpl.XpringException;
+
+import java.math.BigInteger;
+import java.util.Optional;
+
+/**
+ * A client that can create accounts, get accounts, get balances, and send ILP payments on a connector.
+ */
+public class IlpClient {
+    private IlpClientDecorator decoratedClient;
+
+    /**
+     * Initialize a new client with the given options.
+     *
+     */
+    public IlpClient() {
+        this.decoratedClient = new DefaultIlpClient();
```

Comment:
We should generally let folks specify the URLs they want to hit I think. I had put in a PR into the XpringClient hierarchy to do this.